src/interfaces/forms.d.ts

```diff
@@ -78,7 +93,8 @@ export interface IMultiSelectFieldProps extends IMultiSelectProps {
   description?: React.ReactNode;
 }
 
-export interface IInputProps extends React.InputHTMLAttributes<HTMLInputElement> {
+export interface IInputProps
+  extends React.InputHTMLAttributes<HTMLInputElement> {
```

Comment:
could you add `ref` property to types definitions of components with `forwardRef`? Button component is a good example how it should be done

Reply:
only `input` uses forwardRef 😱

Reply:
ohh 😮 we need to add forwardRef to those component in one of next PRs